### ISSUE TITLE
bounding box search

### ISSUE THREAD

**Issue by ouya99** [iotaledger/iota-area-codes#3]
Hi, 

i once came up with a similar idea to iota-area-codes and I wanted to get lets say all neighboring transactions with 10km box around me location (e.g. location is NZHOOK ...), so I want to get for example all TAG TRYTES starting with NZHO....

How would you do such a TAG TRYTE search in IOTA using this module..

```
 var searchNeigbors = function() {
    // location String
    let tryteLat = convertToTryte
    // 0480101103 - LAT 48, LATMIN 1 LNG 11, LNGMIN 3
    let boundingBox = getGeoTryteNeighbours(lat, lng, latmin, lngmin);
    console.log(boundingBox)
```

**Comment by obany** [comment]
This is exactly what we do with the query server https://iota-poc-area-codes.dag.sh/query provide as much of the IAC as you need and then pad with AA.

To implement this we have a layer that monitors the ZMQ stream and any transactions that have an IAC in their tag field we store (splitting into the IAC pairs).
- Monitor https://github.com/iotaledger/iota-area-codes/blob/408a21bf4673998f92741476dc1ac9ac710ce2a5/examples/queryServer/components/zmq.js#L13-L25
- Store https://github.com/iotaledger/iota-area-codes/blob/408a21bf4673998f92741476dc1ac9ac710ce2a5/examples/queryServer/components/db.js#L51-L70

Then to query we just look at the individual pairs within the db https://github.com/iotaledger/iota-area-codes/blob/408a21bf4673998f92741476dc1ac9ac710ce2a5/examples/queryServer/components/db.js#L79-L107

You could of course just capture the transactions you want at the ZMQ stage and do something with them without needing to store them.

**Comment by ouya99** [comment]
Thank you for your great answer!

So the IAC code can be queried only in pairs of TRYTES? (e.g. NZ* , NZHO*, NZHOOK* ) ?

From https://iota-poc-area-codes.dag.sh/query
> The code should be 9 characters long and end in a AA9. To have a wildcard for any other part of the location use padding character pairs of AA.
> Example queries could be NPAAAAAA9 or NPHTAAAA9.

So this sample would then retrieve all TX with TAG = NP* or NPHT* , correct?

Thank you!

**Comment by obany** [comment]
Yes that's exactly right, you always have to have pairs as the pairs are really longtitude/latitude.

and yes again NPHTAAAA9 would retrieve all NPHT*
